pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/OpphoerBarnetilleggAuto.kt

```diff
@@ -273,8 +277,23 @@ object OpphoerBarnetilleggAuto : AutobrevTemplate<OpphoerBarnetilleggAutoDto> {
         includeAttachmentIfNotNull(vedleggMaanedligUfoeretrygdFoerSkatt, maanedligUfoeretrygdFoerSkatt)
         includeAttachment(
             createVedleggOpplysningerBruktIBeregningUT(
-                skalViseMinsteytelse = false,
+                skalViseAvdoed = false,
                 skalViseBarnetillegg = true,
+                skalViseDinUtbetalingFoerSkatt = false,
```

Comment:
Vi kan ikke ha så mange styringer pr brev som skal inkludere vedlegget. Kanskje om du tar ut alle conditions for hvilke brevkoder det gjelder og putter det i en tabell/matrise så kan vi finne hva grupperingene faktisk er istedenfor å kopiere de rett fra extream.

Reply:
Tenker på det.